### PR TITLE
SAK-28084 Make it easy to add anon and auth role

### PR DESCRIPTION
**Pull request by adrianfish**
Make it easy to add anon and auth role to a site using the Site Info
tool.

### CODE REVIEW

File: authz/authz-tool/tool/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java [hunk 1]
```diff
@@ -544,11 +546,6 @@ public void doSave(RunData data)
 	{
 		SessionState state = ((JetspeedRunData) data).getPortletSessionState(((JetspeedRunData) data).getJs_peid());
 
-		if (!"POST".equals(data.getRequest().getMethod())) {
```
Comment (buckett) [inline authz/authz-tool/tool/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java]: I'm not sure this needs to be removed does it?

Comment (adrianfish) [inline authz/authz-tool/tool/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java]: It's not in your Oxford instance, as far as I can tell. One of your commits removed it, so I left it out.

Comment (ottenhoff) [inline authz/authz-tool/tool/src/java/org/sakaiproject/authz/tool/PermissionsHelperAction.java]: I'm with @buckett here: this line deletion is very suspect

**Comment by buckett** [comment]
I've created #419 to add back the HTTP method check. I think it was in our local code as a result of a bad merge.

**Pull request merged**